hi-ml-cpath/testhisto/testhisto/utils/test_viz_utils.py

```diff
@@ -295,13 +294,3 @@ def test_location_selected_tiles(level: int) -> None:
     assert max(tile_xs) <= slide_image.shape[2] // factor
     assert min(tile_ys) >= 0
     assert max(tile_ys) <= slide_image.shape[1] // factor
-
-
-@pytest.mark.parametrize("backend", ["cuCIM", "OpenSlide"])
-@pytest.mark.parametrize("wsi_has_mask", [True, False])
-def test_load_image_dict(wsi_has_mask: bool, backend: str) -> None:
```

Comment:
this test wasn't meaningful, we weren't loading a slide, test_load_slide should cover it netter